.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,7 @@
+<!-- Comment: 
+A great PR typically begins with the line below.
+Replace XXXXX with the numeric part of the issue's id you created on JIRA.
```

Comment:
Perhaps make a mention that a jira-ticket is required for backporting

Reply:
> 
> 
> Perhaps make a mention that a jira-ticket is required for backporting

Done.